<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt5DigitalImageProcessingClass</class>
 <widget class="QMainWindow" name="Qt5DigitalImageProcessingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt5DigitalImageProcessing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" rowspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="TabImgProcess">
       <attribute name="title">
        <string>图像处理</string>
       </attribute>
       <widget class="QLabel" name="labelShow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>731</width>
          <height>421</height>
         </rect>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>260</y>
          <width>201</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="5" column="1">
          <widget class="QCheckBox" name="checkBoxWaterMark">
           <property name="text">
            <string>水印</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonNextImg">
           <property name="text">
            <string>下一张</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButtonTurnRight">
           <property name="text">
            <string>右旋转</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="pushButtonMirrorTB">
           <property name="text">
            <string>上下镜像</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="pushButtonSaveImg">
           <property name="text">
            <string>保存图片</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButtonMirrorLR">
           <property name="text">
            <string>左右镜像</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButtonTurnLeft">
           <property name="text">
            <string>左旋转</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonPreImg">
           <property name="text">
            <string>上一张</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>440</y>
          <width>661</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelOther1">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelOther2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelOther3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButtonGamma">
        <property name="text">
         <string>伽马变换</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButtonMeanFilter">
        <property name="text">
         <string>均值滤波</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonCvtGray">
        <property name="text">
         <string>灰度化</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonEdgeDetection">
        <property name="text">
         <string>边缘检测</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonSelectImage">
        <property name="text">
         <string>选择图片</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonShowOriginImg">
        <property name="text">
         <string>显示原图</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonEdgeDetectAndOriImg">
        <property name="text">
         <string>边缘检测+原图</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelScale">
        <property name="text">
         <string>比例:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelSaturation">
        <property name="text">
         <string>饱和度</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLuminance">
        <property name="text">
         <string>亮度:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="labelSaturationNum">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelThreshold">
        <property name="text">
         <string>二值化</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelContrast">
        <property name="text">
         <string>对比度</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSliderScale_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="labelContrastNum">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelScaleNum">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelThresholdNum">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSliderScale_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSliderScale_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSliderScale">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSliderScale_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelLuminance_2">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="labelColorG">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSliderScale_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSliderScale_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelColorGNum">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QSlider" name="horizontalSliderScale_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelColorB">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelColorR">
        <property name="text">
         <string>   R       </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelColorBNum">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelColorRNum">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionVideo"/>
    <addaction name="actionDock"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuImgFilters">
    <property name="title">
     <string>滤镜</string>
    </property>
    <addaction name="actionCvtGray"/>
    <addaction name="actionMeanFilter"/>
    <addaction name="actionEdgeDetect"/>
    <addaction name="actionGammaAndOriImg"/>
    <addaction name="actionGamma"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuTools_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionLanguage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImgFilters"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
   <addaction name="menuTools_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionVideo"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCvtGray"/>
   <addaction name="separator"/>
   <addaction name="actionGamma"/>
   <addaction name="separator"/>
   <addaction name="actionMeanFilter"/>
   <addaction name="separator"/>
   <addaction name="actionEdgeDetect"/>
   <addaction name="separator"/>
   <addaction name="actionGammaAndOriImg"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/filenew.png</normaloff>:/Images/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
  </action>
  <action name="actionDock">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/zoom-in.png</normaloff>:/Images/images/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>显示Dock</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/picture.png</normaloff>:/Images/images/picture.png</iconset>
   </property>
   <property name="text">
    <string>打开图片(O)</string>
   </property>
   <property name="toolTip">
    <string>打开文件(O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/save.png</normaloff>:/Images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存图片(S)</string>
   </property>
   <property name="toolTip">
    <string>保存图片(S)</string>
   </property>
  </action>
  <action name="actionSaveAnother">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/filesaveas.png</normaloff>:/Images/images/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为...</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/about.png</normaloff>:/Images/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(A)</string>
   </property>
   <property name="toolTip">
    <string>关于(A)</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/help.png</normaloff>:/Images/images/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助(H)</string>
   </property>
   <property name="toolTip">
    <string>帮助(H)</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/zh-en.png</normaloff>:/Images/images/zh-en.png</iconset>
   </property>
   <property name="text">
    <string>切换中英(L)</string>
   </property>
   <property name="toolTip">
    <string>切换中英(L)</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/left.png</normaloff>:/Images/images/left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/right.png</normaloff>:/Images/images/right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/center.png</normaloff>:/Images/images/center.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/video.png</normaloff>:/Images/images/video.png</iconset>
   </property>
   <property name="text">
    <string>打开视频(V)</string>
   </property>
   <property name="toolTip">
    <string>打开视频(V)</string>
   </property>
  </action>
  <action name="actionCvtGray">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/gray.png</normaloff>:/Images/images/gray.png</iconset>
   </property>
   <property name="text">
    <string>灰度化(H)</string>
   </property>
   <property name="toolTip">
    <string>灰度化(H)</string>
   </property>
  </action>
  <action name="actionMeanFilter">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/junzhi.png</normaloff>:/Images/images/junzhi.png</iconset>
   </property>
   <property name="text">
    <string>均值滤波(J)</string>
   </property>
   <property name="toolTip">
    <string>均值滤波(J)</string>
   </property>
  </action>
  <action name="actionEdgeDetect">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/bianyuan.png</normaloff>:/Images/images/bianyuan.png</iconset>
   </property>
   <property name="text">
    <string>边缘检测</string>
   </property>
  </action>
  <action name="actionGammaAndOriImg">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/fuhe.png</normaloff>:/Images/images/fuhe.png</iconset>
   </property>
   <property name="text">
    <string>边缘+原图(Y)</string>
   </property>
   <property name="toolTip">
    <string>边缘+原图(Y)</string>
   </property>
  </action>
  <action name="actionGamma">
   <property name="icon">
    <iconset resource="Qt5DigitalImageProcessing.qrc">
     <normaloff>:/Images/images/gamma.png</normaloff>:/Images/images/gamma.png</iconset>
   </property>
   <property name="text">
    <string>伽马变换(G)</string>
   </property>
   <property name="toolTip">
    <string>伽马变换(G)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Qt5DigitalImageProcessing.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>on_pushButtonSelectImage_clicked()</slot>
 </slots>
</ui>
